src/run.rs

```diff
@@ -670,6 +671,17 @@ fn vacuum(
         });
     }
 
+    // If the user provided the `--min-age` argument, we need to filter out images which are newer than the provided duration.
+    if let Some(duration) = min_age {
+        match (SystemTime::now() - *duration).duration_since(UNIX_EPOCH) {
+            Ok(time_stamp) => {
+                sorted_image_nodes
+                    .retain(|(_, image_node)| image_node.last_used_since_epoch < time_stamp);
```

Comment:
I see that for the `--keep` logic right above this, we log a message (with `debug!`) indicating that the image was ignored. Can we add that here too? Maybe something like:

```rust
sorted_image_nodes.retain(|(_, image_node)| {
    if image_node.last_used_since_epoch < time_stamp {
        debug!(
            "Ignored image {} due to the {} flag.",
            format!("{}:{}", repository_tag.repository, repository_tag.tag).code_str(),
            "--min-age".code_str(),
        );
        return false;
    }

    true
});
```

Reply:
From my knowledge adding {repository}:{tag} is not that straight forward, as `last_used_since_epoch` is set on the `ImageNode`. The ImageNode has an `image_record` that holds the repository info as a vector, which of course means that one repository can have one or more tags associated.

I'm contemplating adding something like this, but I will have to think a bit before settling on a solution.
`format!("{}:*", image_node.image_record.repository_tags[0].repository).code_str(),`.

Reply:
Ah, I see. Something like this then?

```rust
sorted_image_nodes.retain(|(_, image_node)| {
    if image_node.last_used_since_epoch > time_stamp {
        const repository_tag = image_node.image_record.repository_tags[0]; // [ref:at_least_one_repository_tag]
        debug!(
            "Ignored image {} due to the {} flag.",
            format!("{}:{}", repository_tag.repository, repository_tag.tag).code_str(), // [ref:at_least_one_repository_tag]
            "--min-age".code_str(),
        );
        return false;
    }

    true
});
```

Reply:
Found and implemented a solution